Add link to submission for, to publish extensions in the marketplace (#163)
docker/extensions-sdk: docs/extensions/DISTRIBUTION.md

With a given release of Docker Desktop (including some extensions), users should not need to upgrade Docker Desktop in order to obtain new versions of a specific extension. Newer versions of the extension can be released independently of Docker Desktop releases, provided there is no Extension API mismatchs.

Docker Desktop scans the list of published extensions for new versions regularly, and provides some notifications to users when they can upgrade a specific extension.

Users can download and install the newer version of an extension without updating Docker Desktop itself.

> ## Publish your extension in the marketplace

Reviewer: ```suggestion
## Publish your extension in the Marketplace
```

Docker Desktop includes a list of published extensions that have been selected to be displayed in the extension marketplace.

> If you want your extension to be published in the marketplace, you can submit your extension [here](https://www.docker.com/products/extensions/submissions/). We'll review your submission and provide feedback about the extension if needed, in order to publish it and make it available for all Docker Desktop users.

Reviewer: ```suggestion
If you want your extension to be published in the Marketplace, you can submit your extension [here](https://www.docker.com/products/extensions/submissions/). We'll review your submission and provide feedback if changes are needed before your extension is published and made available to all Docker Desktop users.
```
